components/navbar/SideNav.js

```diff
@@ -1,24 +1,27 @@
 import { Disclosure } from "@headlessui/react";
 import ChevronRightIcon from "@heroicons/react/20/solid/ChevronRightIcon";
 import EditOnGitHub from "@components/EditOnGithub";
+import { useRouter } from "next/router";
 
 function classNames(...classes) {
   return classes.filter(Boolean).join(" ");
 }
 
 export default function SideNav({ navigation }) {
+  const { pathname } = useRouter();
   return (
     <ul role="list" className="w-64 flex-none hidden sm:block mt-12">
       <li>
         <ul role="list">
           {navigation.map((item) => (
-            <li key={item.name}>
+            <li key={item.name} className="my-2">
               {!item.children ? (
                 <a
                   href={item.href}
                   className={classNames(
-                    item.current ? "bg-primary-low" : "hover:bg-primary-low",
-                    "group flex gap-x-3 rounded-md p-2 text-md leading-6 font-semibold text-primary-high dark:text-primary-low-medium dark:hover:text-secondary-high hover:text-secondary-high"
+                    item.href == pathname &&
+                      " bg-slate-200 dark:bg-primary-low !text-secondary-high",
```

Comment:
I don't think the prepended space is required as the `classNames` adds a space?

Reply:
Resolved 👍